shared/src/common/enums/exceptions/exception-message.enum.ts

```diff
@@ -6,6 +6,7 @@ enum ExceptionMessage {
   INVALID_GROUP_NAME = 'Group name is invalid.',
   INVALID_GROUP_PERMISSIONS = 'Group permissions are invalid.',
   INVALID_GROUP_USERS = 'Group users are invalid.',
+  PERMISSIONS_ERROR = 'Permissions error occurred.',
```

Comment:
let's use `You don't have permission to access this resource` or something similar